### ISSUE TITLE
addr of extern_var is not loaded

### ISSUE THREAD

**Issue by ducorduck** [lks9/angr#17]
### Description

while writing the test, i've found a bug that the address of `extern_var` in line 25 of `various_variables.c` is not loaded

ex:
```python
import angr

p = angr.Project('various_variables',  load_debug_info = True)
p.kb.nvariables.load_from_dwarf()
main_addr = p.loader.find_symbol("main").rebased_addr
simgr = p.factory.simgr()
simgr.explore(find = main_addr)
main_entry_state = simgr.found[0]

extern_var = main_entry_state.nvariables["extern_var"]
print(extern_var.addr)                                            # this should print the address but instead print None
```

**Comment by lks9** [comment]
Thanks!

From the DWARF standard:

```
2.13.1 Non-Defining Declarations

A debugging information entry that represents a non-defining or otherwise
incomplete declaration of a program entity has a DW_AT_declaration attribute,
which is a flag.
```

Apparently, the DW_AT_location is not defined but there is a chance that it might be defined in some other compilation unit. Apparently, a specific link to that unit is not given in the DWARF. I think we would need to use the symbol resolution of the ELF to find it.

Again, I would say fixing this bug has no priority now.

It would be great if this could be included into your tests. Could you already commit parts of your tests? Thanks in advance!

**Comment by lks9** [comment]
Nevertheless, fixed by 3b8b402.
